packages/design-system/src/components/OcProgressPie/OcProgressPie.vue

```diff
@@ -77,8 +77,8 @@ $default-size: 64px;
 
     &::before,
     &::after {
-      border: calc($default-size / 10) solid var(--oc-color-swatch-passive-default);
-      border-color: var(--oc-color-swatch-passive-default);
+      border: calc($default-size / 10) solid var(--oc-role-secondary);
+      border-color: var(--oc-role-secondary);
```

Comment:
Maybe rather outline role?

Reply:
Weird that we set the `border-color` after it was already set in the `border` attribute...

Reply:
true

Reply:
Checked it, the border is the progress pie fill, so `secondary` is correct.